internal/locate/replica_selector_test.go

```diff
@@ -663,6 +663,28 @@ func TestReplicaReadAccessPathByCase(t *testing.T) {
 		},
 	}
 	s.True(s.runCaseAndCompare(ca))
+
+	s.changeRegionLeader(3)
+	ca = replicaSelectorAccessPathCase{
```

Comment:
We could also cover the `req.ReplicaReadType == kv.ReplicaReadPreferLeader` case.

Reply:
done.